src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/gitlab/GitlabServerPullRequestDecorator.java

```diff
@@ -80,6 +80,9 @@
     public static final String PULLREQUEST_GITLAB_PIPELINE_ID =
             "com.github.mc1arke.sonarqube.plugin.branch.pullrequest.gitlab.pipelineId";
 
+    private static final String NO_COVERAGE_VALUE =
+            "-1";
```

Comment:
The new-line doesn't seem right - this should be on the line above

Reply:
The coding converting wasn't clear to me from the previous lines. Feel free to change it though.